mcmd/commands/make.py

```diff
@@ -56,87 +67,125 @@ def add_arguments(subparsers):
 @command
 def make(args):
     role_name = to_role_name(args.role)
-    group_name = _find_group(role_name)
-    membership = _get_user_group_membership(args.user, group_name)
+    role = _get_role(role_name)
+    user = _get_user(args.user)
+    if role.group:
+        _make_member_of_group_role(user, role)
+    else:
+        _make_member_of_role(role, user)
+
+
+def _make_member_of_group_role(user: User, role: Role):
+    group = role.group
+    membership = _get_group_membership(user, group)
     if membership:
-        if membership.role_name == role_name:
-            io.info('User {} is already {} of group {}'.format(highlight(args.user),
-                                                               highlight(membership.role_label),
-                                                               highlight(group_name)))
+        if membership.role.name == role.name:
+            io.info('User {} is already {} of group {}'.format(highlight(user.username),
+                                                               highlight(role.label),
+                                                               highlight(group.name)))
         else:
             update = ask.confirm(
```

Comment:
how does confirmation work in scripted mode?

Reply:
When running scripts it will still ask for confirmation. 

Some commands have a `--force` argument so that you can use them in automation (for example in a deployment pipeline). I've considered using a `--force` argument for this command but I reckoned that in such scripts you most likely won't change the membership of a user more than once.

What I'd like to do in the future is to have a top-level `--force` argument that will answer yes to all yes/no questions, independent of command. That would solve this problem for automation once and for all (there are some other commands that lack a `--force` as well).